examples/pm25_simpletest.py

```diff
@@ -1,13 +1,15 @@
 """
-Example sketch to connect to PM2.5 sensor with either I2C or UART.
+Example sketch to connect to PM2.5 sensor with I2C.
 """
 
 # pylint: disable=unused-import
 import time
 import board
 import busio
 from digitalio import DigitalInOut, Direction, Pull
-import adafruit_pm25
+from adafruit_pm25.i2c import PM25_I2C
+
+# from adafruit_pm25.uart import PM25_UART
```

Comment:
You may want to move this import under the `# Connect to a PM2.5 sensor over UART` or say "uncomment this if you are using UART`

Reply:
Please move this under `# Connect to a PM2.5 sensor over UART`. I believe that is a better place for it, and it will be more obvious that it should be uncommented for UART.

Reply:
Done, but I start to wonder if it would not be best to have two separate "simpletest", one for I2C and one for UART. Just to keep think more simple.